src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,37 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exceptions.DukeException;
+
```

Comment:
Suggest to have a constant order for import statements. Perhaps, the DukeException statement could be above the import time statements (as in Deadline class)?

```suggestion
import duke.exceptions.DukeException;

import java.time.LocalDate;
import java.time.format.DateTimeFormatter;
import java.time.format.DateTimeParseException;

```